server/src/lexer/pest/python.pest

```diff
@@ -82,7 +82,10 @@ block_comment_ending = { newline_eoi }
 
 // ## Code blocks
 code_block = { code_line+ }
-code_line = _{ (!doc_block ~ logical_line ~ NEWLINE) | (!doc_block ~ not_newline+ ~ EOI) }
+single_line_string = {("\"" ~ (!("\"") ~ ANY)* ~ "\"") | ("'" ~ (!("'") ~ ANY)* ~ "'")}
```

Comment:
This is a good start! First, get this working, then add some tricky extra cases:

``` Python
print("this is\
# not a comment.")
print("this is \" quoted") 
```

Output from running this:
```
this is# not a comment.
this is " quoted
```